DER/DerivativesContracts/Options.rdf

```diff
@@ -254,16 +321,139 @@
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;EquityOption">
+		<rdfs:subClassOf rdf:resource="&fibo-der-sbd-sbd;EquityDerivative"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;Option"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-sbd-sbd;EquityObservable"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-dbt-ex;hasExerciseTerms"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;StockOptionExerciseTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">equity option</rdfs:label>
+		<skos:definition xml:lang="en">Equity options are the most common type of equity derivative. They provide the right, but not the obligation to trade a quantity of stock at a set price at a future time.</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-der-der-opt;ExchangeTradedEquityIndexOption">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;ExchangeTradedOption"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-sbd-sbd;EquityDerivative"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;EquityIndexOptionUnderlying"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-dbt-ex;hasExerciseTerms"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;IndexOptionExerciseTerms"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">exchange traded equity index option</rdfs:label>
+		<skos:definition xml:lang="en">Equity Index options offer the investor an opportunity to either capitalize on an expected market move or to protect holdings in the underlying instruments. The difference is that the underlying instruments are indexes.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">These indexes can reflect the characteristics of either the broad equity market as a whole or specific industry sectors within the marketplace.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
Should explain how this differs from an option on a fund such as SPY as opposed to the real index SPX https://www.thebalance.com/spx-options-vs-spy-options-2536632 

Would indexes include VIX which measures volatility https://www.cboe.com/tradable_products/vix/vix_options/?